### ISSUE TITLE
Flaky-test: BrokerServiceLookupTest. testPartitionTopicLookup

### ISSUE THREAD

**Issue by RobertIndie** [apache/pulsar#14146]
BrokerServiceLookupTest.testPartitionTopicLookup is flaky. It fails sporadically.

Usage tip: To enable automatic navigation to failure message, open the following links with CTRL/CMD-click.
[example failure Mon, 07 Feb 2022 09:47:24 GMT](https://github.com/apache/pulsar/runs/5090628431?check_suite_focus=true#step:9:3696)

```
org.apache.pulsar.client.admin.PulsarAdminException$TimeoutException: java.util.concurrent.TimeoutException
	at org.apache.pulsar.client.admin.internal.BaseResource.sync(BaseResource.java:299)
	at org.apache.pulsar.client.admin.internal.TopicsImpl.deletePartitionedTopic(TopicsImpl.java:445)
	at org.apache.pulsar.client.admin.Topics.deletePartitionedTopic(Topics.java:618)
	at org.apache.pulsar.client.admin.internal.TopicsImpl.deletePartitionedTopic(TopicsImpl.java:435)
	at org.apache.pulsar.client.api.BrokerServiceLookupTest.testPartitionTopicLookup(BrokerServiceLookupTest.java:405)
```

<details>
<summary>Full exception stacktrace</summary>
<code><pre>
Error:  testPartitionTopicLookup(org.apache.pulsar.client.api.BrokerServiceLookupTest)  Time elapsed: 62.177 s  <<< FAILURE!
org.apache.pulsar.client.admin.PulsarAdminException$TimeoutException: java.util.concurrent.TimeoutException
	at org.apache.pulsar.client.admin.internal.BaseResource.sync(BaseResource.java:299)
	at org.apache.pulsar.client.admin.internal.TopicsImpl.deletePartitionedTopic(TopicsImpl.java:445)
	at org.apache.pulsar.client.admin.Topics.deletePartitionedTopic(Topics.java:618)
	at org.apache.pulsar.client.admin.internal.TopicsImpl.deletePartitionedTopic(TopicsImpl.java:435)
	at org.apache.pulsar.client.api.BrokerServiceLookupTest.testPartitionTopicLookup(BrokerServiceLookupTest.java:405)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
	at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)
	at org.apache.pulsar.client.admin.internal.BaseResource.sync(BaseResource.java:292)
	... 16 more
</pre></code>
</details>

**Comment by Demogorgon314** [comment]
It was fixed by PR #14060.

**Comment by lhotari** [comment]
duplicates #14046
